doc.go

```diff
@@ -6,7 +6,7 @@ Package ear implements an EAT attestation result format based on the
 information model defined in
 https://datatracker.ietf.org/doc/draft-ietf-rats-ar4si/
 
-Construction
+# Construction
```

Comment:
this doesn't render as you may expect (try `godoc` locally)

Reply:
Argh. The linter was complaining about that one..